news/get-authors.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+
```

Comment:
I didn't see a general Copyright claim on Anitya - I also recommend adding a copyright claim on this file.

Reply:
Same as above.